src/amo/components/SearchResults.js

```diff
@@ -29,10 +30,20 @@ class SearchResults extends React.Component {
     } = this.props;
     const { query } = filters;
 
+    let loadingMessage;
     let messageText;
 
     if (loading) {
-      messageText = i18n.gettext('Searching...');
+      loadingMessage = (
+        <div className="LoadingIndicator-container"
+          ref={(ref) => { this.LoadingIndicatorContainer = ref; }}>
+          <LoadingIndicator>
+            <span className="visually-hidden">
```

Comment:
It seems like it might be common to include hidden text like this. I think this would be nicer as:
````js
<LoadingIndicator altText={i18n.gettext('Searching…')} />
````